gui-apps/ags: new package, add 1.8.0

DerpDays opened the pull request.

Jamim commented on gui-apps/ags/ags-1.8.0.ebuild:
```diff
@@ -0,0 +1,88 @@
+
+pkg_postinst() {
+	elog "To learn on how to use ags, please read"
+	elog "https://github.com/Aylur/ags/wiki/configuration"
```
That page says
> The wiki has been migrated to [aylur.github.io/ags-docs/](https://aylur.github.io/ags-docs/)

```suggestion
	elog "https://aylur.github.io/ags-docs/"
```

DerpDays commented:
Yep, will do later today, forgot to check before I made the PR

DerpDays commented:
I've cleaned it up a bit, though I'm not 100% sure if `pam` is a required dependency, though i *think* it should be fine.
The new commits also added a pam file since users need one for using `Utils.authenticate`.
Let me know if there's anything I should change or remove!

*Best regards!*

Jamim reviewed (changes requested):
Thank you for addressing the issues, @DerpDays!
Now I have a few more suggestions.

DerpDays commented:
I've addressed the suggestions you have mentioned, feel free to take another look

Best regards!
